packages/conjure-client/src/fetchBridge/fetchBridge.ts

```diff
@@ -190,7 +190,11 @@ export class FetchBridge implements IHttpApiBridge {
         return path;
     }
 
-    private buildQueryString(data: { [key: string]: any }) {
+    private buildQueryString(data: any) {
+        if (data == null) {
```

Comment:
this should be sufficient since `Object.keys()` seems to coerce all other types correctly